services/frontend/src/components/DetailsForm/DetailsForm.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import {TextField} from "@material-ui/core";
+import "./DetailsForm.scss";
+
+const DetailsForm = (formFieldData) => (
```

Comment:
Instead of spreading the `formFieldData` over the `DetailsForm` such that all of the props are _just_ the form field data, I'd suggest instead passing the data as a single prop, so that future users of this component aren't confused when they try to add new props to it and shit goes sideways.

Reply:
Sure, done.